esp-hal-common/src/system.rs

```diff
@@ -186,6 +194,69 @@ impl PeripheralClockControl {
     }
 }
 
+#[cfg(esp32c6)]
+impl PeripheralClockControl {
+    /// Enables and resets the given peripheral
+    pub fn enable(&mut self, peripheral: Peripheral) {
+        let system = unsafe { &*SystemPeripheral::PTR };
+
+        match peripheral {
+            Peripheral::Spi2 => {
+                system.spi2_conf.modify(|_, w| w.spi2_clk_en().set_bit());
+                system.spi2_conf.modify(|_, w| w.spi2_rst_en().clear_bit());
+            }
+            Peripheral::I2cExt0 => {
+                // FIXME
```

Comment:
Seems like this should be addressed before we merge.

Reply:
@bjoernQ would you mind looking at this? I can take a look but can't test anything. I think that the registers I expected weren't present or something, but they may be hiding elsewhere or be named differently.